docs/develop/use-sso-to-get-office-signed-in-user-token.md

```diff
@@ -61,19 +61,22 @@ To use SSO with Office, you need to create an app registration in the Azure port
    > [!NOTE]
    > The domain part of the **Scope** name displayed just below the text field should automatically match the Application ID URI that you set earlier, with `/access_as_user` appended to the end; for example, `api://localhost:6789/c6c1f32b-5e55-4997-881a-753cc1d563b7/access_as_user`.
 
-1. In the **Authorized client applications** section, you identify the applications that you want to authorize to your add-in's web application. Each of the following IDs needs to be pre-authorized.
+1. In the **Authorized client applications** section, enter the following ID to pre-authorize all Microsoft Office application endpoints.
 
-   - `d3590ed6-52b3-4102-aeff-aad2292ab01c` (Microsoft Office)
-   - `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` (Microsoft Office)
-   - `57fb890c-0dab-4253-a5e0-7188c88b2bb4` (Office on the web)
-   - `08e18876-6177-487e-b8b5-cf950c1e598c` (Office on the web)
-   - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Outlook on the web)
+   - `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` (All Microsoft Office application endpoints)
 
-   For each ID, take these steps:
+    > [!NOTE]
+    > The `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` ID pre-authorizes all the following IDs. You can enter the following IDs individually if you prefer.
+    >
+    > - `d3590ed6-52b3-4102-aeff-aad2292ab01c` (Microsoft Office)
+    > - `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` (Microsoft Office)
+    > - `93d53678-613d-4013-afc1-62e9e444a0a5` (Office on the web)
+    > - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Outlook on the web)
```

Comment:
Well, we don't want to list ea5 here too, because that would make the other 3 redundant and confuse things. I think we want to list just the other three and say something about why they might take this option.

```suggestion
    > The `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e` ID pre-authorizes Office on all the following platforms. Alternatively, you can enter a proper subset of the following IDs if for any reason you want to deny authorization to Office on some platforms. Just leave out the IDs of the platforms from which you want to withhold authorization. Users of your add-in on those platforms will not be able to call your Web APIs, but other functionality in your add-in will still work.
    >
    > - `d3590ed6-52b3-4102-aeff-aad2292ab01c` (Microsoft Office)
    > - `93d53678-613d-4013-afc1-62e9e444a0a5` (Office on the web)
    > - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Outlook on the web)
```